Revise analysis example for current API

rocky opened the pull request.

Commit:
Revise analysis example for current API

fgimenez commented on example/analysis:
```diff
@@ -1,48 +1,44 @@
 #!/usr/bin/env node
 
 "use strict"
+var fs = require('fs')
```
This should be a `const`

fgimenez commented on example/analysis:
```diff
@@ -1,48 +1,44 @@
   Example code starts here.....
 ***********************************/
 
+var options = {
```
`const` here too

fgimenez commented on example/analysis:
```diff
@@ -1,48 +1,44 @@
 const client = new armlet.Client(options)
 
-client.analyze({ bytecode: bytecode, timeout: timeout }) // parameter "timeout" is optional
+client.analyze(options) // parameter "timeout" is optional
```
The comment about the timeout param is stale now.

fgimenez reviewed (dismissed):
LGTM, a few small nitpicks.

Commit:
Incorporate Federico's excellent suggestions

Commit:
Add JSON from...

https://capturetheether.com/challenges/miscellaneous/token-bank/
https://capturetheether.com/challenges/math/token-whale/

fgimenez reviewed (approved):
LGTM
